cron/hourly/70-cancel-subscriptions-for-cancelled-orders.ts

```diff
@@ -28,24 +29,38 @@ const getHostFromOrder = async order => {
   return models.Collective.findByPk(transactions[0].HostCollectiveId);
 };
 
+const getOrderCancelationReason = (collective, order, orderHost) => {
+  if (!order.Tier) {
+    return `Order tier deleted`;
```

Comment:
:no_entry_sign: That's not a safe assumption: when you make a "Custom contribution" (e.g. https://opencollective.com/babel/donate) orders are created without a tier (`TierId = null`). We need to find a better way of doing this, one could be:

```suggestion
  if (order.TierId && !order.Tier) {
    return `Order tier deleted`;
```

Note that we have an open discussion about changing this in https://github.com/opencollective/opencollective/issues/3055, but with so many things relying on this pattern it's unlikely to change anytime soon.